docs/react/toggling-devices.mdx

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 4
+---
+
+# Toggling devices
+
+Fishjam Cloud provides functionality to control media device streams more dynamically. This includes both soft control, such as muting, and hard control, like physically turning the device on or off.
+
+## Soft Mode - muting and unmuting
+
+Soft Mode allows you to enable or disable the media stream without affecting the device's operational status.  
+If the soft mode is disabled, it starts the device, enables the media stream, and starts (or resumes) streaming.  
+When it's enabled, it disables the media stream and pauses streaming but does not stop the device.  
+In the case when the device is off, it starts the device, enables the media stream, and starts (or resumes) streaming.
+
+### Usage example
+
+Use this function to programmatically toggle the mute state based on user interactions or other application logic.
+
+```tsx
+import { useCamera } from "@fishjam-cloud/react-client";
+
+function CameraControl() {
+  const camera = useCamera();
+
+  return (
+    <button onClick={() => camera.toggleMute()}>Toggle camera mute</button>
```

Comment:
nit: I would swap the examples. I would use 'camera' in 'toggleDevice' and 'microphone' in 'toggleMute' for convenience as I described in this comment.

Reply:
Counterargument: I think showing both functions on the same device is beneficial, as it doesn't lead to an assumption that you can only toggle the camera and mute the microphone, which coincidentally matches the colloquial usage of these words.

Reply:
leaving as it is for now